.github/workflows/android_arm64.yml

```diff
@@ -5,7 +5,7 @@ on:
   merge_group:
   push:
     branches:
-      - master
+      # - master
```

Comment:
the same for releases,, right?
I assume merge queue is enabled for future releases branches

Reply:
It's not enabled yet, since it's possible only for branch protection rules without wildcards, so once a new release branch appears, we'll need to configure it separately. Plus, we have older releases, so maybe it makes sense to leave it as-is now?